Source/Core/Core/PowerPC/Jit64/Jit_LoadStore.cpp

```diff
@@ -304,7 +304,9 @@ void Jit64::dcbz(UGeckoInstruction inst)
     AND(32, R(RSCRATCH), Imm32(~31));
   }
 
-  if (MSR.DR)
+  bool emit_fast_path = MSR.DR && m_jit.jo.fastmem_arena;
```

Comment:
Can this be `jo.fastmem` instead as `fastmem` checks for `fastmem_arena` in the base class?

Reply:
I tried to keep this separate. jo.fastmem_arena might be turned on without the exception handlers. The checks for jo.fastmem requires both.